package.json

```diff
@@ -20,7 +20,7 @@
     "pre-commit": "lint-staged"
   },
   "lint-staged": {
-    "*.js": [
+    "*.{js|ts}": [
```

Comment:
https://github.com/eslint/rewrite/blob/c3fea625b74d62a059e4a16a76fb025519076fc7/package.json#L27

We also have this where we run prettier? Does that mean for `.ts` prettier will run twice now?

Reply:
Yes. 

I think this PR change may be unnecessary since we're already linting `.ts` files.
Ref https://github.com/eslint/rewrite/pull/22#discussion_r1598320289

Reply:
Good catch! Added `.ts` to the second entry to ensure it won't be run through prettier twice.

Reply:
> Added `.ts` to the second entry to ensure it won't be run through prettier twice.

Maybe this change isn't pushed?